netjsonconfig/backends/openwrt/converters/wireless.py

```diff
@@ -269,3 +314,16 @@ def __get_netjson_interface(self, wifi):
             if interface['name'] == wifi['ifname']:
                 interface['type'] = 'wireless'
                 return interface
+
+    def to_netjson_clean(self, intermediate_data):
+        result = super().to_netjson_clean(intermediate_data)
+        self._set_wireless_interface_bridges(self.backend._intermediate_copy)
+        for index, interface in enumerate(result):
+            try:
+                bridges = self._wireless_interface_bridges[interface['ifname']]
+            except KeyError:
+                continue
+            else:
+                if bridges == interface.get('network', '').split(' '):
+                    del result[index]['network']
+        return result
```

Comment:
The old tests are failing because they did not expect the `network` option to be removed automatically. I have not updated the tests deliberately at this moment. I will update them once we are sure that we want to continue on this path.